src/app/project/ontology/resource-class-info/resource-class-info.component.ts

```diff
@@ -514,4 +515,32 @@ OFFSET 0`;
         window.open(doSearchRoute, '_blank');
     }
 
+    createResourceInstance(iri: string, label: string) {
+        let projectIri: string;
+        // get project iri
+        this._cache.get(this.projectCode).subscribe(
+            (res: ReadProject) => {
+                projectIri = res.id;
+            }
+        );
+
+
+        const dialogConfig: MatDialogConfig = {
+            width: '840px',
+            maxHeight: '80vh',
+            position: {
+                top: '112px'
+            },
+            data: { id: iri, mode: 'createResource', project: projectIri, title: label,  subtitle: 'Set the property values of the resource' },
+            disableClose: true
+        };
+
+        const dialogRef = this._dialog.open(
```

Comment:
If you don't have any future plans for anything happening after the dialog is closed, you don't need to assign this to a `const` and can remove the logic below for doing something after the dialog is closed. But it's fine if we keep it as well. Maybe we'll need it in the future.

Reply:
Good point. Thanks. But I think I'll keep it for the moment. Just copied from other components.